backend/src/routes/api/k8s/pass-through.ts

```diff
@@ -81,6 +52,10 @@ export const safeURLPassThrough = <T extends K8sResourceCommon>(
             }`,
           );
           throw { code: parsedData.code, response: parsedData };
+        } else if (data.method.toLowerCase() === 'delete') {
+          // TODO: Remove when bug is fixed - https://issues.redhat.com/browse/RHCLOUD-28058
+          // Return a non-status object upon successful delete
+          return {} as K8sStatus;
```

Comment:
You can get 4.0.1 to avoid this hack.

Reply:
right. I didn't revisit the branch code itself because it was previously reviewed.